pkg/synthesis/tests_expected_output/k8s_resources/Example1c_NoHint/policies.yaml

```
@@ -1,11 +1,34 @@
 apiVersion: networking.k8s.io/v1
 kind: NetworkPolicy
+metadata:
+    annotations:
+        description: Network Policy To Allow Access To DNS Server
+        nsx-id: dns-rule-id
```

this is not related to nsx-id
```suggestion
        nsx-id:  none
```